apps/client/app/purchase-orders/page.tsx

```diff
@@ -1,8 +1,112 @@
+import Link from 'next/link';
+import moment from 'moment';
+import { sort } from 'fast-sort';
+import Currency from '../components/Currency';
+
+interface PurchaseOrderLineItem {
```

Comment:
Did you explore using DTOs and transformers to map the database entities to client usable objects? How could this have changed the implementation, simplified or increase the complexity of the code?